bin/charliecloud.h

```diff
@@ -1,15 +1,47 @@
 /* Copyright © Los Alamos National Security, LLC, and others. */
 
-/* Test some result: if not zero, exit with an error. This is a macro so we
-   have access to the file and line number.
-
-   Note: This macro is sometimes used for things other than system calls. If
-   errno is not set when the program aborts, the resulting error message will
-   say "error: Success", which is very confusing. Thus, make sure that users
-   won't see those. */
-#define TRY(x) if (x) fatal_errno(NULL, __FILE__, __LINE__)
-#define TRX(x, msg) if (x) fatal_errno(msg, __FILE__, __LINE__)
-
-void fatal(char * fmt, ...);
-void fatal_errno(char * msg, char * file, int line);
+/* Test some value, and if it's not what we expect, exit with an error. These
+   are macros so we have access to the file and line number.
+
+                     verify x is true (non-zero) and print:
```

Comment:
I think it's more straightforward to use "otherwise", i.e. `verify x is true (non-zero), otherwise print`.